core/modules/parsers/wikiparser/rules/filteredtranscludeblock.js

```diff
@@ -42,21 +52,21 @@ exports.parse = function() {
 	var node = {
 		type: "list",
 		attributes: {
-			filter: {type: "string", value: filter}
+			filter: {type: "string", value: filter, start: filterStart, end: filterEnd},
```

Comment:
It might be more consistent and user friendly to use attributeStart and attributeEnd property names for all the attributes, this applies to all wikirules.

Reply:
You mean use `{type: "string", value: filter, filterStart, filterEnd}` here?

Reply:
@linonetwo I must have misread the code this morning, this code is fine and needs no further changes, and this comment can be ignored.